design/index.md

```diff
@@ -42,9 +42,95 @@ Other tools included in the Creative Cloud suite that may be useful include:
+
+## Social Media, Marketing & PR
+
+It's important we understand what we communicate, to who, how, and when. This sets out our principles for social media, marketing, and PR communications.
+
+### Content Types
+
+Content should be published to the appropriate channels, the content we create is as follows.
+
+#### Release
+
+Major releases of FlowForge, the timing of these releases are governed by our [development cadence](https://flowforge.com/handbook/development/#cadence).
+
+#### Patch
+
+Patch between releases, these will happen as needed and may be at short notice.
+
+#### Newsletter
+
+Monthly newsletter published at the start of each calendar month covering news from the prior month.
+
+#### How To
+
+Technical guides, published once per week.
+
+#### Planned Downtime
+
+As soon as we are aware of planned downtime we should publish the information, that should be at least one week before the downtime.
+
+#### Outage
+
+If we experience unplanned downtime we should communicate in near real time to keep our users and stakeholders up to date with the situation.
+
+#### Corporate Comms
+
+Major company news such as investment rounds should be communicated at the time agreed with the relevant parties involved. We strive to make this information public as soon as practical and effective.
+
+### Content Channels
+
+Each content type has channels which are appropriate, this table sets out which content should be posted on each channel.
+
+|Content          |Blog  |Mailing List |Twitter  |Youtube  |LinkedIn |Reddit |Node-RED Slack|
+|---              |---   |---          |---      |---      |---      |---    |---           |
+|Release          |*     |             |*        |*        |*        |*      |*             |  
+|Patch            |      |             |*        |         |         |*      |*             |   
+|Newsletter       |*     |*            |*        |         |*        |*      |*             |
+|How To           |      |*            |*        |         |*        | *     |*             |  
+|Planned Downtime |      |*            |*        |         |*        |*      |*             |  
+|Outage           |      |             |         |         |         |       |              |
+|Corporate Comms  |*     |             |*        |         |*        |       |              |
+
+### Channel Content Guidance
+
+#### Blog
+
+Content should be eloquent and an appropriate length to communicate the message.
+
+#### Mailing List
+
+Content should be eloquent and an appropriate length to communicate the message.
+
+#### Twitter
+
+Keep it short, Tweets should be factual, informal and have a call to action (a link) where a user can read more.
+
+#### Youtube
+
+Videos should:
+
+- Include opening and closing idents
+- Be clear in what they are communicating.
+- Have an appropriate thumbnail.
+- Have a call to action in the description.
+- Be publicly listed.
+
```

Comment:
Further details will be found in the (to be reviewed/merged) `/handbook/media` section